stock/templates/registered/corputils.html

```diff
@@ -0,0 +1,132 @@
+{% extends "public/base.html" %}
+{% load bootstrap %}
+{% load corputils_extras %}
```

Comment:
if corputils is in your installed apps setting, it should be found. However, as you noted it will run without it, it was something I used in my first iteration and forgot to remove...